test/local_test_ckpt_index_file.py

```diff
@@ -6,10 +6,7 @@
 import json
 import os
 
-from pippy.hf._SaveModule import (
-    _save_index,
-    _save_checkpoint,
-)
+from pippy.hf._SaveModule import save_checkpoint
```

Comment:
Also, I think we can update this test name to reflect what we are testing. 
local_test_save_load_checkpoint.py